tremc

```diff
@@ -46,59 +32,330 @@ from textwrap import wrap
+                                         epilog="Positional arguments are passed to transmission-remote. Use -- to separate from %(prog)s arguments")
+        parser.add_argument("-v", "--version", action="version", version="%(prog)s " + self.VERSION,
+                            help="Show version number and supported Transmission versions.")
+        parser.add_argument("-c", "--connect", action="store", dest="connection", default="",
+                            help="Point to the server using pattern [username:password@]host[:port]/[path]")
+        parser.add_argument("-s", "--ssl", action="store_true", dest="ssl", default=False,
+                            help="Connect to Transmission using SSL.")
+        parser.add_argument("-f", "--config", action="store", dest="configfile", default=default_config_path,
+                            help="Path to configuration file.")
+        parser.add_argument("--create-config", action="store_true", default=False,
+                            help="Create configuration file CONFIGFILE with default values.")
+        parser.add_argument("-l", "--list-actions", action="store_true", dest='listactions', default=False,
+                            help="List available actions for key mapping.")
+        parser.add_argument("-k", "--list-keys", action="store_true", dest='listkeys', default=False,
+                            help="List available key names for key mapping.")
+        parser.add_argument("-n", "--netrc", action="store_true", dest="use_netrc", default=False,
+                            help="Get authentication info from your ~/.netrc file.")
+        parser.add_argument("-X", "--skip-version-check", "--permissive", action="store_true", dest="PERMISSIVE", default=False,
+                            help="Proceed even if the running transmission daemon seems incompatible, or the terminal is too small.")
+        parser.add_argument("-p", "--profile", action="store", dest="profile",
+                            help="Select profile to use.")
+        parser.add_argument("-r", "--reverse-dns", action="store_true", dest="rdns", default=False,
+                            help="Toggle reverse DNS peers addresses.")
+        parser.add_argument("-d", "--debug", action="store", dest="DEBUG", nargs='?', default=False,
+                            help="Enable debugging messages.")
+        parser.add_argument('transmissionremote_args', nargs='*', metavar='A',
+                            help="Torrent files to add using transmission-remote")
+        cmd_args = parser.parse_args()
+
+        for i in vars(cmd_args).keys():
+            setattr(self, i, vars(cmd_args)[i])
+
+        if self.DEBUG is None:
+            self.DEBUG = True
+            self.debug_file = sys.stderr
+        elif isinstance(self.DEBUG, str):
+            try:
+                self.debug_file = open(self.DEBUG, "a", buffering=1)
+            except OSError:
```

Comment:
Why are we setting logs through a file like this instead of using the [logging](https://docs.python.org/3.8/library/logging.html) module?

Reply:
It might not be the most "python" way, but when I only need a single simple function, I generally prefer to avoid importing a module. I am not sure the program will be more readable using logging module.

Reply:
Fair Enough. I presume if we ever introduce more flexible logging (more than just debug, including warning, info etc.) we'll switch to the module then. It should be a straightforward change from what you've implemented so this seems okay. :+1:.